streampipes-client/src/main/java/org/apache/streampipes/client/api/AdminApi.java

```diff
@@ -98,4 +112,16 @@ private StreamPipesApiPath getFunctionsPath() {
   private StreamPipesApiPath getDeleteFunctionPath(String functionId) {
     return getFunctionsPath().addToPath(functionId);
   }
+
+  private StreamPipesApiPath getAdapterMigrationPath() {
+    return StreamPipesApiPath
+            .fromBaseApiPath()
+            .addToPath("migrations/adapter");
```

Comment:
I suggest to use `addToPath` for every URL part: `addToPath("migrations").addToPath("adapter") and then to reuse the migrations path in a constant